src/category_theory/equivalence.lean

```diff
@@ -187,6 +188,25 @@ by { dsimp [inv_fun_id_assoc], tidy }
   (inv_fun_id_assoc e F).inv.app X = F.map (e.counit_inv.app X) :=
 by { dsimp [inv_fun_id_assoc], tidy }
 
+section
+omit 𝒟 ℰ
+
+-- There's of course a monoid structure on `C ≌ C`,
+-- but let's not encourage using it.
+-- The power structure is nevertheless useful
+
+/-- Powers of an auto-equivalence. -/
+def pow (e : C ≌ C) : ℤ → (C ≌ C)
+| (int.of_nat 0) := equivalence.refl
+| (int.of_nat 1) := e
+| (int.of_nat (n+2)) := e.trans (pow (int.of_nat (n+1)))
+| (int.neg_succ_of_nat 0) := e.symm
+| (int.neg_succ_of_nat (n+1)) := e.symm.trans (pow (int.neg_succ_of_nat n))
+
+instance : has_pow (C ≌ C) ℤ := ⟨pow⟩
```

Comment:
Should this come with simp-lemmas or canonical-isom-defs for the cases `n in {-1, 0, 1}`, and similarly for `(n+1)` etc?

Reply:
I added the simp lemmas for `-1, 0, 1`. I don't think we're ready yet to express the power laws like `(e^a).trans (e^b) \iso e^(a+b)`, as we haven't even specified the category structure on equivalences, let alone the monoidal category structure on autoequivalences!